invenio_app_ils/ui/src/invenio_app_ils/store.js

```diff
@@ -30,6 +34,8 @@ const rootReducer = combineReducers({
   userDetails: userDetailsReducer,
   patronLoans: patronLoansReducer,
   availableItems: availableItemsReducer,
+  locations: locationsListReducer,
```

Comment:
`locationsListReducer`, `internalLocationReducer` one plural with List postfix the other singular, I hope we can all feel where this is going. Once more I would bleed to death arguing to kill the plural.

Reply:
I will kill the plural and re push it!